main/ser2net/APKBUILD

```diff
@@ -12,56 +12,33 @@ depends_dev=""
 makedepends="$depends_dev linux-headers"
 install=""
 subpackages="$pkgname-doc"
-source="https://sourceforge.net/projects/ser2net/files/ser2net/ser2net-$pkgver.tar.gz
-	TIOCSRS485.patch
-	${pkgname}.confd
-	${pkgname}.initd"
-
-_builddir="$srcdir"/ser2net-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-}
+source="https://sourceforge.net/projects/$pkgname/files/$pkgname/$pkgname-$pkgver.tar.gz
```

Comment:
please set the `source` variable to: `https://sourceforge.net/projects/ser2net/files/ser2net/$pkgname-$pkgver.tar.gz`

Reply:
for curiosity, what the rationale behind this request? For the comment about url field, I get it, it's convenient for a copy/paste. But for the source field, we already use variables, so what's the reason to use it only for the tarball name?

Reply:
It makes it easier to select the link up to the variables `https://sourceforge.net/projects/ser2net/files/ser2net/` and click go to, instead of copy-pasting the link, replacing the variables and then clicking ok, go etc.
It helps if you want to take a quick look at the distfiles.

Reply:
Thanks for giving some more input on your rationale.